js/src/setup-mc/setup-stepper/setup-accounts/index.js

```diff
@@ -181,8 +171,14 @@ const SetupAccounts = ( props ) => {
 						{ __( 'Continue', 'google-listings-and-ads' ) }
 					</AppButton>
 				</StepContentActions>
-				<Faqs />
 			</StepContentFooter>
+			<Section
+				className="gla-google-mc-account-section"
```

Comment:
Just noting that this classname should probably be changed here and in the relevant styles [here](https://github.com/woocommerce/google-listings-and-ads/blob/c1d9c8224d531344fd3b8b21a3c0863575bd52f5/js/src/setup-mc/setup-stepper/setup-accounts/index.scss#L5), but it's also likely that this text will end up needing to be moved somewhere else before #2509 is completed, so I don't see that as a blocker.

Reply:
Agree that renaming would be more appropriate.

Reply:
👍🏻 Added [a note](https://github.com/woocommerce/google-listings-and-ads/issues/2509#issuecomment-2338941336) in the tracking ticket to remind us to come back to this.

Reply:
Renamed in https://github.com/woocommerce/google-listings-and-ads/pull/2634/commits/cc2440a6d45d234ed095689af297aeaf0337247a.